### PR TITLE
Audit table fixes

### PR DESCRIPTION
**Pull request by andrewpatto**
Corrections to audit table to match other table formats (basically removing most formatting and letting daisy ui defaults)
Default filter for top level audit page is "user" only
Rolled duration column underneath time (duration is rarely filled in and is unlikely for sorting/filtering)
(temp) Removed possibility of going to detail page (need to work out how it works with breadcrumbs etc)

### CODE REVIEW

**Comment by andrewpatto** [comment]
Screenshot 2023-05-05 at 2 55 53 pm

**Comment by andrewpatto** [comment]
Changed the language throughout to refer to only Audit Events and Audit Event

File: application/backend/src/api/api-auth-routes.ts [hunk 1]
```diff
@@ -168,7 +168,11 @@ export const callbackRoutes = async (
     const authUser = await userService.upsertUserForLogin(
       idClaims.sub,
       idClaims.name,
-      idClaims.email
+      idClaims.email,
+      {
+        ip: request.ip,
+        hostname: request.hostname,
+      }
```
Comment (andrewpatto) [inline application/backend/src/api/api-auth-routes.ts:175]: To better have examples of audit entries with details - we mind as well always log extra details of logins

**Review by williamputraintan — APPROVED** [review]

**Review by mmalenic — APPROVED** [review]
Looks good! I don't think anything has been broken.

**Pull request merged**